ui/perfherder/alerts/Assignee.jsx

```diff
@@ -10,7 +10,7 @@ export default class Assignee extends React.Component {
     this.state = {
       assigneeUsername,
       inEditMode: false,
-      newAssigneeUsername: null,
+      newAssigneeUsername: "",
```

Comment:
I think you need to pull the latest changes from the upstream repo and then `git merge master`.
I assume this change is not intended.